lib/miq_automation_engine/service_models/miq_ae_service_physical_server.rb

```diff
@@ -0,0 +1,23 @@
+module MiqAeMethodService
+  class MiqAeServicePhysicalServer < MiqAeServiceModelBase
+  	expose :ext_management_system, :association => true
+  	expose :physical_servers,      :association => true
+
+  	def turn_on_loc_led
+  		object_send(:turn_on_loc_led)
+  	end
+
+  	def turn_off_loc_led
+  		object_send(:turn_off_loc_led)
+  	end
+
+  	def power_on
+  		object_send(:power_on)
+  	end
+
+  	def power_off
```

Comment:
These methods just pass the method to the instance so you can do them as a single line `expose` for each:

```ruby
  expose turn_on_loc_led
  expose turn_off_loc_led
  ...
```